### ISSUE TITLE
Legacy network doesn't work at all

### ISSUE THREAD

**Issue by chaollapark** [koii-network/ezsandbox#80]
TLDR
1. I tried out blocks, accounts, transactions, programs, and tokens (that exist);
2. The button works but the information doesn't appear (only the icon as in the example image);
3. Also the Cluster stats, supply, inspector buttons work but they don't loan any information at all;

I'd say it's a major bug, because if the legacy testnet is still up there are still projects or people who need the legacy button.

Let me know!!! Hopefully I get the bounty!!!! 

![Screen Shot 2024-07-08 at 5 04 18 PM](https://github.com/koii-network/ezsandbox/assets/133244975/9ac8a532-e7cf-4518-bfa8-674e9aad4f96)

**Comment by labrocadabro** [comment]
Legacy testnet is deprecated; I'm putting in an issue internally to remove it from the explorer. Thank you!

**Comment by chaollapark** [comment]
Thank you for the quick response. 

Could you clarify if there's a process for reporting user experience issues like outdated UI elements? Also, is there documentation available that outlines which features are currently supported vs deprecated? This could help contributors like myself focus on relevant areas for improvement.

**Comment by labrocadabro** [comment]
All issues should be reported here, regardless of their nature. However, some of them (like this one) have to be dealt with in other repos. All contributions are useful, but most people will probably focus on the EZSandbox instructions and any issues that arise during the task development process.
